docs/source/contributing/code-of-conduct.rst

```diff
@@ -0,0 +1,75 @@
+***************************************
+Code of Conduct
+***************************************
+
+Our Pledge
+==========
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+Our Standards
+=============
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+Our Responsibilities
+====================
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+Scope
+=====
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+Enforcement
+===========
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting squad-federated-learning@owkin.com, which is a shared team inbox. If the incident involves someone who receives that shared inbox, you can contact an individual maintainer. All
```

Comment:
Don't we have a mail more specific to this use case ? (asking @RomainGoussault also). Cause this one is quite dependent on the current organisation and might become obsolete quite fast...

Reply:
hum agree, and it should not be an owkin address but a LFAI address.
Another way to put it is to report to the code owners? The email address of the code owner should be accessible?

Reply:
No there no way with github to have this functionnality... I agree for the LFAI address, or maybe ask the IT if we could have a support@substra.com or something like that ? Cause it should be an address we're actually using! And if coming from the IT, we can link the mail to our owkin adress

Reply:
I am checking if we have an LFAI address. I will keep you posted shortly.

Reply:
Sorry I don't have an answer yet. 
Let's not block the merging for this issue though, we will change the address alter.